instagram/src/PostsPage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+// import PostContainer from './index.js';
+import SearchBar from './components/SearchBar/index.js';
+import PostContainer from './components/PostContainer';
+import data from './dummy-data.js';
+import './index.css';
+
+// const PostsPage = props => {
+//     return (
+// 		<div>
+//             return (
+//                 <PostContainer data={this.state.data}/>
+//             )
+// 		</div>
+//     )
+// }
+
+// export default PostsPage;
+
+
+
+class PostsPage extends React.Component {
+	constructor() {
+		super();
+		this.state = {
+			data: [],
+		};
+	}
+	
+	componentDidMount() {
+		this.setState({ data: data});
```

Comment:
With ES6 if the key and the value are the same, you can shorten in

For example `this.setState({ data: data});` can be shortened to `this.setState({ data });`